drivers/pds/lib/utils.go

```diff
@@ -301,14 +306,14 @@ func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName string) (string, str
 	if !istargetclusterAvailable {
 		log.Fatalf("Target cluster is not available for the account/tenant")
```

Comment:
Do not `log.Fatalf()`, return like for all the above errors, stick to the same return methods

Reply:
Done